Jenkinsfile.promote

```diff
@@ -43,6 +43,7 @@ pipeline {
         OPENSHIFT_CREDS_KEY = "OPENSHIFT_CREDS"
 
         BOT_CREDENTIALS_ID="bsig-gh-bot"
+        GITHUB_REPO = "kogito-cloud-operator"
```

Comment:
where is that one used ?

Reply:
For `github-release`: 
```
-r, --repo           Github repo (required if $GITHUB_REPO not set)
```

I'll add it as a comment.